cEP-0025.md

```diff
@@ -0,0 +1,169 @@
+# Integration of pyflakes-enhanced-AST into coala
+
+| Metadata |                                                 |
+| -------- | ----------------------------------------------- |
+| cEP      | 22                                              |
+| Version  | 1.0                                             |
+| Title    | Integration of pyflakes-enhanced-AST into coala |
+| Authors  | Ankit Joshi <mailto:ajankit2304@gmail.com>      |
+| Status   | Proposed                                        |
+| Type     | Feature                                         |
+
+## Abstract
+
+This document describes how to create a metabear to integrate
+pyflakes-enhanced-AST into coala and demonstrates its use by
+creating a simple dependent bear.
+
+## Introduction
+
+Flake8 is the most commonly used linter framework when it comes to static
+code analysis for python. It basically wraps around the tools pyflakes,
+pycodestyle and mccabe scripts because each of those projects
```

Comment:
> because each ...

That because is a bit strange.
it wraps then because they are useful tools with difference interfaces.
flakes v2 wasnt just a pycodestyle wrapper . It also used pycodestyle engine , plugin system, config system.  It was a hack on pycodestyle.  pyflakes doesnt provide error codes. Flake8 does, and that is why ppl used it , and built new plugins.